### PR TITLE
Add PR template

### PR DESCRIPTION
**Pull request by froboy**
**Related Issue/Ticket:**

No issue. I know https://github.com/ymcatwincities/openy/blob/8.x-2.x/.github/PULL_REQUEST_TEMPLATE.md exists, but I thought we could use something a little smaller here.

## To Test:

- [ ] Merge this PR
- [ ] Create a new PR
- [ ] Check out the great template!

### CODE REVIEW

**Review by podarok — DISMISSED** [review]

**Review by hamrant — DISMISSED** [review]

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+**Related Issue/Ticket:**
+
+(Replace this line with a link to the related GitHub issue on [ymcatwincities/openy_gated_content](https://github.com/ymcatwincities/openy_gated_content/issues) or your own ticketing system)
+
+## To Test:
```
Comment (anpolimus) [inline .github/pull_request_template.md:5]:
```suggestion
## Steps to test:
```

**Review by anpolimus — CHANGES_REQUESTED** [review]
@froboy thank you for this PR. We really need this!
Since we've focused on it, lets make the template complete:
- please add upgrade path section
  We have to inform the developer that the upgrade path for the existed project is very important
- we have to add note that for frontend fixes it should be tested across all 3 themes: Rose, Lily, Carnation
- please add section and inform developer that he/she should add Need review tag to start review.

**Comment by froboy** [comment]
@anpolimus I modified your suggestion a little, as only the Open Y team has privileges to add tags here, but based on my conversations with the team, it sounds like anyone outside of OY/5J/IX who makes PRs should be notifying folks in Slack, so I added that.

**Review by hamrant — APPROVED** [review]

**Review by anpolimus — APPROVED** [review]

**Comment by anpolimus** [comment]
Thank you! Now, our template looks amazing!

**Pull request merged**